src/components/Content/Content.vue

```diff
@@ -85,6 +85,8 @@ export default {
 	display: flex;
 	padding-top: 50px;
 	min-height: 100%;
+	background: url("/src/assets/kamil-porembinski-clouds.jpg");
```

Comment:
to be removed once server puts the image behind every app content page

Reply:
Let me push that to my server pr as well then

Reply:
Available now as --image-main-background which will also adapt to theming https://github.com/nextcloud/server/pull/33610